app/editor/src/features/content/form/TimeLogTable.tsx

```diff
@@ -1,46 +1,39 @@
 import { useFormikContext } from 'formik';
 import React from 'react';
 import { useLookup } from 'store/hooks';
-import { GridTable, IContentModel, ITimeTrackingModel, Row } from 'tno-core';
+import { GridTable, ITimeTrackingModel, Row } from 'tno-core';
 
+import { ITimeTrackingForm } from './components/time-log/interfaces';
 import { timeLogColumns } from './constants';
+import { IContentForm } from './interfaces';
 import * as styled from './styled';
 
-export interface ITimeLogTableProps {
```

Comment:
When using formik it's much better to use the context to get data, don't pass through properties because they won't update state